frontend/src/components/regions/CardDetailsModal/Presentation.js

```diff
@@ -53,7 +55,7 @@ const TopicProgressDetails = ({ topicProgress, reviews }) => {
   );
 };
 
-const CardBasicDetails = ({ card }) => {
+const CardBasicDetails = ({ card, authUser }) => {
```

Comment:
You've added an extra argument to this function.  It would be good to make sure that existing code doesn't break as a result.

Where is CardBasicDetails used in the code? Does it recieve the correct props? 

Your story will look of, but the code wont actually work in context.